qemu/tests/cfg/stop_cont_vm_during_virtiofs.cfg

```diff
@@ -0,0 +1,38 @@
+- stop_cont_vm_during_virtiofs:
```

Comment:
Could you change test case name to virtio_fs_subtest_during_io, as we also have reboot, shutdown vm tests and so on.